google/data_source_stackdriver_ip_ranges.go

```diff
@@ -0,0 +1,55 @@
+package google
+
+import (
+	"fmt"
+	"log"
+	"sort"
+	"time"
+
+	"github.com/hashicorp/terraform/helper/schema"
+	monitoring "google.golang.org/api/monitoring/v3"
+)
+
+func dataSourceStackdriverIpRanges() *schema.Resource {
+	return &schema.Resource{
+		Read: dataSourceStackdriverIpRangesRead,
+
+		Schema: map[string]*schema.Schema{
+			"cidr_blocks": {
```

Comment:
Likewise, the field actually returned from the API is also called `uptimeCheckIps`. I'm not 100% opposed to renaming them, but there has to be a good reason behind it since (at least in my opinion) it's easier to find things if they keep the same name everywhere (terraform, gcloud, rest api, etc.)

Reply:
`cidr_blocks` seems something I've seen on AWS resources.